mlflow/transformers.py

```diff
@@ -1664,20 +1664,18 @@ def trim_input(data_in, data_out):
             if (
                 data_out.startswith(data_in + "\n\n")
                 and flavor_config[_INSTANCE_TYPE_KEY] in self._supported_custom_generator_types
+                and not include_prompt
```

Comment:
```suggestion
                not include_prompt
                and data_out.startswith(data_in + "\n\n")
                and flavor_config[_INSTANCE_TYPE_KEY] in self._supported_custom_generator_types

```

We can check the flag first

Reply:
nice catch on the short circuit :) TY!